homework01/artur/vector.h

```diff
@@ -0,0 +1,25 @@
+#include <iostream>
```

Comment:
Is this needed?

Reply:
It is for the ostream operator overload, right?